### DISCUSSION TITLE
Light css variable overrides dark theme

### DISCUSSION THREAD

**Discussion by edgarriba** [pradyunsg/furo#251] [Potential Issue] [answered]
Hi! I'm trying to override only the light css theme as follows:
```python
    "light_css_variables": {
        "color-sidebar-background": "#3980F5",
        "color-sidebar-caption-text": "white",
        "color-sidebar-link-text--top-level": "#cccccc",
    },
```
the light theme is fine, however the dark theme is overridden. 

![image](https://user-images.githubusercontent.com/5157099/133932994-271320c8-5973-48bc-91fb-184e660eab3c.png)

![image](https://user-images.githubusercontent.com/5157099/133933003-cdd29a3c-75d7-4e1a-b624-64c4e8285c99.png)

Is there anything I'm missing here ? thanks in advance !

**Comment by pradyunsg** [comment, marked as answer]
Hiya! Please read https://pradyunsg.me/furo/customisation/colors/#how-light-and-dark-mode-work.

**Reply by edgarriba** [reply]
@pradyunsg thanks ! I had to override (by hand) the the dark style according to the css theme. Would be nice that it's done automatically though in the future. Because, in case anyone change dark style, will have to change hardcoded colors.

Thanks!
